src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -230,4 +230,70 @@ if (!product) {
 ### Server Endpoints
 
 A server endpoint, also known as an **API route**, is a special function exported from a `.js` or `.ts` file within the `src/pages/` folder. 
-The function takes an [endpoint context](/en/reference/api-reference/#endpoint-context) and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response). A powerful feature of SSR, API routes are able to securely execute code on the server. To learn more, see our [Endpoints Guide](/en/core-concepts/endpoints/#server-endpoints-api-routes).
\ No newline at end of file
+The function takes an [endpoint context](/en/reference/api-reference/#endpoint-context) and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response). A powerful feature of SSR, API routes are able to securely execute code on the server. To learn more, see our [Endpoints Guide](/en/core-concepts/endpoints/#server-endpoints-api-routes).
+
+### Streaming
+
+Astro enables HTTP streaming for all pages that are not prerendered. This means that your page's content will be sent incrementally to the page as it resolves async functions and requests.
+
+You can `await` promises in the frontmatter of your page, or you can move those requests down into Astro components the page renders. The page will then send down the HTML it's capable of sending down without waiting for those async components, and then it will start rendering the components in the order they are found in your page's HTML one by one.
```

Comment:
```suggestion
To control the order in which your page renders content, you can adjust where you `await` promises. 

When you `await` promises in the frontmatter of your page, your initial page rendering waits for this async data. Moving those requests into separate Astro components means that your page around these components can now render without waiting. Then, it will start rendering the async components in the order they are found in your page's HTML, one by one.
```
K, just a rough edit to start and I might have lost some nuance (and might have got something wrong!), but I think something like this hits the use-case harder up front?